### ISSUE TITLE
[BUG] G502 mouse making Mobile App Sensor Multiple times when detached (using KVM Switch)

### ISSUE THREAD

**Issue by mxp83** [joshuar/go-hass-agent#374]
**Go Hass Agent Version**
v10.5.1

**Describe the bug**
In Home Assistant, inside mobile app, my G502 has multiple unknown sensors due to when I switch my KVM to PC while my linux box is on.  The last valid state is still shown, and when reconnected adds a new sensor with _2 (ascending number) at the end.

**To Reproduce**
Steps to reproduce the behaviour:
1. Have KBM switch with G502 wireless dongle attached connected to 2 machines (in my case a win 11, and fedora bazzite on the other)
2. Switch input source (from Linux to PC).  Wait a few mins, switch back.
3. Go to Home Assistant integration and look at sensors.
4. See image.

**Expected behaviour**
Maintain the original sensor.

**Screenshots**
[Imgur Link](https://imgur.com/PpLs0M2)

**Logs**
I currently don't have a log because I the linux machine is off (Steam Deck) and not connected at the moment.  Will upload later.

**Desktop (please complete the following information):**
 - OS: Bazzite 41 (Linux Fedora)
 - Distribution Linux Fedora 41
 - Version 41

**Additional context**
KVM switch is a USB3 switch which is also powered externally by an adapter.  Single button on the switch alternates the source.  The port used on the switch for the Logitech G502 is USB3.  The G502 is currently using the Input Remapper software to enable Logitech sidebuttons.

**Comment by joshuar** [comment]
Oh dear, that's no good. Do all the duplicate sensors continue to update and have the same value? Or do only the new (or old) sensors update?

**Comment by mxp83** [comment]
Yes and no.  I say both because the old mouse becomes Unknown (until it becomes unavailable, guessing due to old states cleanup) and a new '_number' shows up with the current.  See [here](https://imgur.com/a/HWhSW4S).  The new ones are prefixed 'deckard'.

It should probably be noted that these sensors are not in recorder, since I don't need to know the batt values or wattage.

**Comment by Vaskyy** [comment]
I can also confirm this at least for the last time i looked. Always wondered why i have soo many mouses..
Which was a few weeks ago.
Right now i made a fresh install and will look into this if my G502 sensors will also duplicate themselves.
I also use it with a super old KVM Switch to change from Win11 and Raspberry Pi and back.
The KVM Switch has even VGA on it, im just using the USB ports tho.

**Comment by Vaskyy** [comment]
Still a problem:
<img width="203" height="882" alt="Image" src="https://github.com/user-attachments/assets/304b0ed2-b5af-4eec-988f-eb0c829a9165" />
